nodes/nomos-node/src/bridges/waku.rs

```diff
@@ -10,57 +10,14 @@ use nomos_mempool::network::adapters::waku::{
     WAKU_CARNOT_PUB_SUB_TOPIC, WAKU_CARNOT_TX_CONTENT_TOPIC,
 };
 use nomos_mempool::MempoolMsg;
+use nomos_network::backends::libp2p;
```

Comment:
libp2p shouldn't be included in `waku.rs` file?

Reply:
oh f**k, autoimport...